pipeline/assets/gtfs.py

```diff
@@ -75,8 +74,7 @@ def reload_pgbouncer_databases(import_op):
 
 @graph_asset(
     group_name='gtfs',
-    auto_materialize_policy=AutoMaterializePolicy.eager(),
-    freshness_policy=FreshnessPolicy(maximum_lag_minutes=60 * 24, cron_schedule='0 1 * * *'),
+    automation_condition=(AutomationCondition.on_cron('0 1 * * *') & ~AutomationCondition.in_progress() | AutomationCondition.eager()),
```

Comment:
Do you know what will happen to the "maximum lag" behaviour?

Although I can easily lookup the precedence rules of binary operators, I think it won't hurt to add a comment explaining the logic in boolean terms & plain language.